modules/classifications/auxiliaries/define/reponse.go

```diff
@@ -47,7 +46,7 @@ func GetClassificationIDFromResponse(AuxiliaryResponse helpers.AuxiliaryResponse
 		if value.IsSuccessful() {
 			return value.ClassificationID, nil
 		} else {
-			return value.ClassificationID, value.GetError()
+			return nil, value.GetError()
```

Comment:
required for a scenario where one tries to define a new auxiliary but one already exists so need to return the already existing classification id